packages/ui/components/form-core/test/validate/lion-validation-feedback.test.js

```diff
@@ -83,12 +83,12 @@ describe('lion-validation-feedback', () => {
     await el.updateComplete;
 
     const validationFeedbackType = el.shadowRoot?.querySelector('.validation-feedback__type');
-    expect(validationFeedbackType?.textContent?.trim()).to.equal('Error');
+    expect(validationFeedbackType?.textContent?.trim()).to.equal('Error,');
 
     el.feedbackData = [{ message: 'hello', type: 'info', validator: new AlwaysInvalid() }];
     await el.updateComplete;
 
-    expect(validationFeedbackType?.textContent?.trim()).to.equal('Info');
+    expect(validationFeedbackType?.textContent?.trim()).to.equal('Info,');
```

Comment:
Is `,` (comma) better than `:` (colon) ?
(In an older iteration of a design system we worked on we also had the sr-only "Error:" prefix)

Reply:
I haven't tried that one. Will see whats the difference.

Reply:
No difference. Either way is fine with me.